src/satellite/oplog.ts

```diff
@@ -131,6 +159,12 @@ export const operationsToTableChanges = (
       for (const [key, value] of Object.entries(entryChanges.changes)) {
         existing.changes[key] = value
       }
+      if (entryChanges.optype == 'DELETE') {
+        existing.tag = null
+      } else {
+        existing.tag = genTag(new Date(entry.timestamp))
+      }
+      existing.clearTags = entryChanges.clearTags
```

Comment:
If I understand correctly, when we compute a snapshot, we compute the clear set per entry and here we are taking the latest clear set from all entries.
I think we could have the following case:
t=[1], c=[]
t=[], c=[1]
t=[2], c=[]

And we would lose the clear of tag 1. Please provide a counter example if this is not a correct interpretation.

Reply:
fixed